src/shared/port-provider.js

```diff
@@ -74,7 +73,7 @@ export class PortProvider {
      * Guest frames in particular may send duplicate requests (see comments in
      * PortFinder).
      *
-     * @type {WeakMap<Window, Set<Channel>>}
+     * @type {WeakMap<Window, Set<Frame>>}
```

Comment:
The comment for this field needs an update. It currently reads "Map of source frame to channels". It would need to be something like "Map of source frame to destination frame role" or similar.